recipes-core/initscripts/files/mdev.conf

```diff
@@ -0,0 +1,41 @@
+console 0:0 0600
+ttyHS0 0:20 660
+ttyHSL0 0:20 660
+ttyHSL1 0:0 600
+cpu_dma_latency 0:0 0660
+fb0:0 44 0660
+full 0:0 0666
+initctl 0:0 0600
+ircomm[0-9].* 0:20 0660
+kmem 0:15 0640
+kmsg 0:0 0660
+log 0:0 0666
+loop[0-9].* 0:6 0640
+mem 0:15 0640
+network_latency 0:0 0660
+network_throughput 0:0 0660
+null 0:0 0666
+port 0:15 0640
+ptmx 0:5 0666
+ram[0-9].* 0:6 0640
+random 0:0 0666
+sda 0:6 0640
+tty 0:5 0666
+ttyUSB[0-9]+ 0:0 0620 */etc/mdev/ttyUSB.sh
+tty.* 0:0 0620
+urandom 0:0 0666
+usbdev.* 0:0 0660 */etc/mdev/usb.sh
+vcs.* 0:5 0660
+zero 0:0 0666
+
+snd/pcm.* 0:0 0660
+snd/control.* 0:0 0660
+snd/timer 0:0 0660
+snd/seq 0:0 0660
+snd/mini.* 0:00 0660
+
+input/event.* 0:0 0660 @/etc/mdev/find-touchscreen.sh
+input/mice 0:0 0660
+input/mouse.* 0:0 0660
+
+tun[0-9]* 0:0 0660 =net/
```

Comment:
Where did all these settings come from?

Reply:
It's a copy of mdev.conf from meta-swi except with one line added for ttyUSB.  Not the prettiest solution, but I'm not aware of a better one.

Reply:
Should we be just patching in the settings that we need to add?

Reply:
That's an alternative.  It's likely to break if they change the file in meta-swi since the patch context and line numbers would change.  I'm doing a bit of reading to see if there's a better way.  Digging a little deeper, it looks like meta-swi's `mdev.conf` is already a copy-then-modify of the one from `poky/meta/recipes-core/busybox/files`.